NSS-Day-Cohort-66/rare-api-eager-miracles
# Ndw postreaction view

## Added PostReaction View with list and create method
## Added reaction images to reactions fixture
## Added delete method to Reactions View

### To Test

- Pull down `ndw-postreaction-view`
-  Start the debugger and open Postman
- Create GET Request with url `http://localhost:8000/postreactions`
- Make sure there is an Authorization Header with viable Token

Expected Payload
- `200 OK`
- `[
    {
        "id": 1,
        "user_id": 1,
        "post_id": 1,
        "reaction_id": 1
    },
    {
        "id": 2,
        "user_id": 2,
        "post_id": 2,
        "reaction_id": 1
    }
]`

### To Test Create Method

- In Postman select POST method with url `http://localhost:8000/postreactions`
- Make sure you have Authorization Header with viable Token
- create a Body with JSON/raw format with example object
`{"post": 2,
"reaction": 3}`
- Expected payload
- `201 Created`
- `{
    "user_id": 2,
    "post_id": 2,
    "reaction_id": 3
}`
